python/lsst/dax/imgserv/config/config.json

```diff
@@ -0,0 +1,10 @@
+{
+        "DEBUG": true,
+	"DB_CONFIG": "~/.lsst/dbAuth-dbServ.ini",
```

Comment:
These are among the core reasons why config handling has not been dealt with till now.  I think we can separate at least 2 major categories of config data - 1) public/general 2) private/sensitive.  My intention is all config data checked into GH should be in the 1st category.  Where to put them (nginx, docker image, webserv, imgserv) is another open issue that needs a further discussion.